src/position.h

```diff
@@ -442,13 +445,24 @@ inline Square Position::royal_king(Color c, Bitboard kings) const {
   for (File f = FILE_A; f <= FILE_H; ++f)
   {
       if (kings & file_bb(f))
-          return lsb(kings & file_bb(f)); // use msb for black to get symmetrical variant
+#ifdef TWOKINGSSYMMETRIC
+          return is_two_kings_symmetric() && c == BLACK ? msb(kings & file_bb(f))
```

Comment:
Could we use convenience function `backmost_sq(c, kings & file_bb(f))`?  I just discovered this while looking for something else.

Reply:
Yes, that makes sense. So something like
```
ifdef TWOKINGSSYMMETRIC
  return backmost_sq(is_two_kings_symmetric() ? c : WHITE, kings & file_bb(f))
#else
  return backmost_sq(WHITE, kings & file_bb(f))
#endif
```
?

Reply:
Yes, that looks like the cleanest way to write this.

Reply:
It it possible to download variant SF and play TwoKing locally on my machine or do it need more testing (not stable enough) ?

Reply:
I think it's stable since after compiling with `debug=yes` (and in `position.cpp`, with `Fast = false`), `./stockfish bench all` (which tests numerous conditions) completes successfully.

Reply:
How to compile latest version?

Reply:
I have created #445 to address this.